runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -89,20 +91,24 @@ end
 
 --- Converts a raw token list to a list of highlight ranges used by the on_win callback
 ---
+---@param data any[]
```

Comment:
```suggestion
---@param data integer[]
```

Reply:
This causes a warning on line 132:

```lua
    local delta_line = data[i]
    line = line and line + delta_line or delta_line
```

Reply:
Ah fixed, `line` is an integer, not a string.